.github/workflows/api-json.yml

```diff
@@ -0,0 +1,42 @@
+name: Generating API Config JSON
+
+on:
+  push:
+    paths:
+    - 'config/v3/**'
+
+jobs:
+  generate_api_json:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:  
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Installing directory tree dependency
+        run: npm i -g directory-tree
+      - name: Generate JSON file with Javascript
+        uses: actions/github-script@v6
+        with:
+          script: |
+            const fs = require("fs");
+            const dirTree = require("directory-tree");
+            const tree = dirTree('./config/v3', {extensions:/\.json$/});
+            fs.writeFile('./api-data/config-tree.json', JSON.stringify(tree, null, 2), function (err) {
+                if (err) throw err;
+                console.log('Saved!');
+            });
+      - name: Commit terminology
+        run: |
+          git add config-tree.json
+          git commit -am "Config tree json file updated"
```

Comment:
I don't see adding the user name? I wonder how will it look like when you trigger it - whether it will be "nobody" like the backend does? Can you manually trigger workflow and see?

Reply:
Will do! Time to test this.